### PR TITLE
add: options to display icons for several HUDs

### PR DESCRIPTION
**Pull request by MisterCheezeCake**
Added options to display icons for the Health, Mana, Defense, and Effective Health HUD.

### CODE REVIEW

File: src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java [hunk 1]
```diff
@@ -783,7 +783,7 @@ public void drawText(Feature feature, float scale, Minecraft mc, ButtonLocation
 
         switch (feature) {
             case MANA_TEXT:
-                text = TextUtils.formatNumber(getAttribute(Attribute.MANA)) + "/" + TextUtils.formatNumber(getAttribute(Attribute.MAX_MANA));
+                text = TextUtils.formatNumber(getAttribute(Attribute.MANA)) + "/" + TextUtils.formatNumber(getAttribute(Attribute.MAX_MANA)) + (main.getConfigValues().isEnabled(Feature.MANA_TEXT_ICON) ? "✎" : "");
```
Comment (Fix3dll) [inline src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java:786]: Please prefer to use `Feature.MANA_TEXT_ICON.isEnabled()`

File: src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java [hunk 2]
```diff
@@ -803,6 +803,10 @@ public void drawText(Feature feature, float scale, Minecraft mc, ButtonLocation
                 } else {
                     text += TextUtils.formatNumber(getAttribute(Attribute.MAX_HEALTH));
                 }
+                if (main.getConfigValues().isEnabled(Feature.HEALTH_TEXT_ICON)) {
```
Comment (Fix3dll) [inline src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java:806]: ^ `Feature.HEALTH_TEXT_ICON.isEnabled()`

File: src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java [hunk 3]
```diff
@@ -816,7 +820,7 @@ public void drawText(Feature feature, float scale, Minecraft mc, ButtonLocation
 
             case DEFENCE_TEXT:
                 if (onRift) return;
-                text = TextUtils.formatNumber(getAttribute(Attribute.DEFENCE));
+                text = TextUtils.formatNumber(getAttribute(Attribute.DEFENCE)) + (main.getConfigValues().isEnabled(Feature.DEFENCE_TEXT_ICON) ? "❈" : "");
```
Comment (Fix3dll) [inline src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java:823]: ^

File: src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java [hunk 4]
```diff
@@ -828,7 +832,7 @@ public void drawText(Feature feature, float scale, Minecraft mc, ButtonLocation
 
             case EFFECTIVE_HEALTH_TEXT:
                 if (onRift) return;
-                text = TextUtils.formatNumber(Math.round(getAttribute(Attribute.HEALTH) * (1 + getAttribute(Attribute.DEFENCE) / 100F)));
+                text = TextUtils.formatNumber(Math.round(getAttribute(Attribute.HEALTH) * (1 + getAttribute(Attribute.DEFENCE) / 100F))) + (main.getConfigValues().isEnabled(Feature.EFFECTIVE_HEALTH_TEXT) ? "❤" : "");
```
Comment (Fix3dll) [inline src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java:835]: ^

File: src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java [hunk 5]
```diff
@@ -1662,7 +1666,7 @@ public void drawText(Feature feature, float scale, Minecraft mc, ButtonLocation
 
                     color = main.getConfigValues().getColor(feature);
                     DrawUtils.drawText(
-                            "/" + TextUtils.formatNumber(getAttribute(Attribute.MAX_HEALTH))
+                            "/" + TextUtils.formatNumber(getAttribute(Attribute.MAX_HEALTH)) + (main.getConfigValues().isEnabled(Feature.HEALTH_TEXT_ICON) ? "❤" : "")
```
Comment (Fix3dll) [inline src/main/java/codes/biscuit/skyblockaddons/listeners/RenderListener.java:1669]: ^

**Review by Fix3dll — CHANGES_REQUESTED** [review]
I would appreciate it if you could make these small changes.

**Comment by MisterCheezeCake** [comment]
Requested changes made

**Review by Fix3dll — APPROVED** [review]
LGTM

**Pull request merged**